packetbeat/docs/faq.asciidoc

```diff
@@ -4,9 +4,33 @@
 This section contains frequently asked questions about Packetbeat. Also check out the
 https://discuss.elastic.co/c/beats/packetbeat[Packetbeat discussion forum].
 
+[[client-server-fields-empty]]
 === Why are the client_server and server fields empty?
 
-The `client_server` and `server` fiels will be empty if you have not configured Packetbeat
+The `client_server` and `server` fields are empty when Packetbeat is not configured
 to capture information about the network topology.
 
-To capture information about the network topology, you need to set the `save_topology` configuration option to true and make sure that you are sending the output to Elasticsearch.
\ No newline at end of file
+To capture information about the network topology, set the `save_topology` configuration option to true and make sure that you are sending the output to Elasticsearch.
+
+[[dashboard-fields-incorrect]]
+=== Why is the dashboard in Kibana breaking up my data fields incorrectly?
+
+The index template might not be loaded correctly. See <<packetbeat-template>>.
+
+[[packetbeat-mirror-ports]]
+=== Why doesn’t Packetbeat see any packets when using mirror ports?
+
+The interface needs to be set to promiscuous mode. Run the following command:
+
+["source","sh",subs="attributes,callouts"]
+----------------------------------------------------------------------
+ip link set enp5s0f1 promisc on
+----------------------------------------------------------------------
+
+[[packetbeat-missing-transactions]]
+=== Why is Packetbeat missing long running transactions?
+
+Packetbeat has an internal timeout that it uses to time out transactions and TCP connections
+when no packets have been seen for a long time. 
```

Comment:
@urso Just want to confirm that you are talking about the `timeout` option here and not the `transaction_timeout` option.